lib/helpers/combineURLs.js

```diff
@@ -8,5 +8,8 @@
  * @returns {string} The combined URL
  */
 module.exports = function combineURLs(baseURL, relativeURL) {
-  return baseURL.replace(/\/+$/, '') + '/' + relativeURL.replace(/^\/+/, '');
+  var end = relativeURL.replace(/^\/+/, '');
```

Comment:
If `relativeURL` is `'/'`, it will not be added, right? For example:

```js
combineURLs('http://test.com/foo', '/'); // => 'http://test.com/foo'
```

I would suggest to append `relativeURL` unless it's falsy (`''`, `undefined`, `null`). Thoughts?